Scheduler/v7/maintel_fbs_sit_block_t345.yaml

```diff
@@ -0,0 +1,47 @@
+maintel:
+  driver_type: feature_scheduler
+  mode: ADVANCE
+  startup_type: COLD
+  startup_database: /home/saluser/rubin_sim_data/fbs_observation_database.sql
+  instrument_name: CCCamera
+  models:
+    observatory_model:
+      camera:
+        filter_max_changes_burst_num: 1000000
+        filter_max_changes_avg_num: 30000
+      optics_loop_corr:
+        tel_optics_cl_alt_limit:
+          - 0
+          - 60
+          - 90
+      park:
+        filter_position: r_03
+  driver_configuration:
+    parameters:
+      night_boundary: -10.0
+    stop_tracking_observing_script_name: maintel/stop_tracking.py
+  feature_scheduler_driver_configuration:
+    observation_database_name: /home/saluser/rubin_sim_data/fbs_observation_database.sql
```

Comment:
This should be `/home/saluser/rubin_sim_data/fbs_observation_database_maintel.sql`